olive/passes/onnx/nvmo_quantization.py

```diff
@@ -0,0 +1,129 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from copy import deepcopy
+from enum import Enum
+from pathlib import Path
+from typing import Any, Callable, Dict, Union
+
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import OliveModelHandler
+from olive.model.utils import resolve_onnx_path
+from olive.passes import Pass
+from olive.passes.onnx.common import model_proto_to_olive_model
+from olive.passes.pass_config import ParamCategory, PassConfigParam
+from olive.strategy.search_parameter import Categorical
+
+logger = logging.getLogger(__name__)
+
+
+# static quantization specific config
+_dataloader_config = {
+    "batch_size": PassConfigParam(
+        type_=int,
+        default_value=1,
+        description="Batch size for calibration.",
+    ),
+    "calib_size": PassConfigParam(
+        type_=int,
+        default_value=64,
+        description="Calibration data size.",
+    ),
+    "dataloader_func": PassConfigParam(
```

Comment:
nitpick: btw, it is fine not to add data_config here. I am planning to implent a standalone data container to unify dataloader_func and data_config.

Reply:
I think this was a follow up to another discussion about data configs. There is no change needed from your side at this point.